ts/features/itwallet/presentation/remote/screens/ItwRemoteFailureScreen.tsx

```diff
@@ -224,6 +226,32 @@ const ContentView = ({ failure }: ContentViewProps) => {
             }
           };
         }
+        case RemoteFailureType.INVALID_CREDENTIALS_STATUS: {
+          const { invalidCredentials } = failure.reason;
+          const count = invalidCredentials.length;
+          return {
+            title: I18n.t(`${i18nNs}.invalidCredentialsScreen.title`, {
+              count,
+              credentialName: getCredentialNameFromType(invalidCredentials[0]),
```

Comment:
In the case of multiple invalid credentials, only the first invalid credential will be displayed, not both. Is this case desired?

Reply:
It is correct. In case of multiple credentials the copy is "Non puoi continuare con la richiesta perché alcuni documenti non risultano validi". So `credentialName` is actually ignored in the i18n interpolation.